DomainContent/Rust/DJ-Tools/DJ_EndPoint_Particle_Server.js

```diff
@@ -0,0 +1,115 @@
+(function () {
+    // Init
+    var _entityID,
+        directionArray,
+        NORMAL = 0,
+        REVERSE = 1,
+        event;
+
+
+    // Polyfill
+    Script.require(Script.resolvePath("./Polyfills.js"))();
+    
+    // Collections
+    var emitOff = { emitRate: 0 },
+        emitOn = { emitRate: 1000 };
+
+    // Helper Functions
```

Comment:
Since this is used in a few places in this PR, I'd suggest moving them into a utility.js file to avoid duplicating the code where possible

Reply:
done